frontend/src/tabs/nfts/NFTs.css

```diff
@@ -5,6 +5,11 @@
 
   transition: all 0.5s;
 }
+@media screen and (max-width: 767px) {
```

Comment:
It would be best to move media queries to here in app.js, so the same queries can be used for all styles :
https://github.com/keep-starknet-strange/art-peace/blob/e3cf53f80e3ea869ca2f84ac2f575606c9d761c2/frontend/src/App.js#L25